dominion/dominion-frontend/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
Again, single licence per repo, hence `HURUmap-apps/LICENSE` is enough (unless it doesn't exist).